docs/source/quickstart.rst

```diff
@@ -153,23 +153,41 @@ When you run the example, it outputs an MLflow run ID for that experiment. If yo
 ``mlflow ui``, you will also see that the run saved a ``model`` folder containing an ``MLmodel``
 description file and a pickled scikit-learn model. You can pass the run ID and the path of the model
 within the artifacts directory (here "model") to various tools. For example, MLflow includes a
-simple REST server for scikit-learn models:
+simple REST server for python-based models:
 
 .. code:: bash
 
-    mlflow sklearn serve -r <RUN_ID> -m model
+    mlflow pyfunc serve -r <RUN_ID> -m model
 
 .. note::
 
     By default the server runs on port 5000. If that port is already in use, use the `--port` option to
-    specify a different port. For example: ``mlflow sklearn serve --port 1234 -r <RUN_ID> -m model``
+    specify a different port. For example: ``mlflow pyfunc serve --port 1234 -r <RUN_ID> -m model``
 
-Once you have started the server, you can pass it some sample data with ``curl`` and see the
-predictions:
+Once you have started the server, you can pass it some sample data and see the
+predictions. The server accepts the following data formats as inputs:
+
+    - JSON-serialized Pandas Dataframes in the `split` orientation.
+      For example, `data = pandas_df.to_json(orient='split')`. This format is specified using a
+      `Content-Type` request header value of `application/json.pandas.split.oriented`.
+
+    - JSON-serialized Pandas Dataframes in the `records` orientation. This format is specified using
+      a `Content-Type` request header value of `application/json`. **THIS FORMAT IS DEPRECATED. It
+      is not guaranteed to preserve column ordering and will be removed in the next release of
+      MLflow**.
+
+    - CSV-serialized Pandas Dataframes. For example, `data = pandas_df.to_csv()`. This format is
+      specified using a `Content-Type` request header value of `text/csv`.
+
+For more information about serializing Pandas Dataframes, see
+https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_json.html
+
+The following example uses ``curl`` to send a JSON-serialized Pandas Dataframe with the `split`
+orientation to the pyfunc server:
 
 .. code:: bash
 
-    curl -d '[{"x": 1}, {"x": -1}]' -H 'Content-Type: application/json' -X POST localhost:5000/invocations
+    curl -d '{"columns":["x"],"index":[0,1],"data":[[1],[-1]]}' -H 'Content-Type: application/json.pandas.split.oriented' -X POST localhost:5000/invocations
```

Comment:
Maybe add some spaces to make this more readable.

Reply:
Added spacing and an explanation of the optional `index` field (which I removed from this example).